src/parser/priest/discipline/modules/Abilities.ts

```diff
@@ -60,7 +60,7 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.POWER_WORD_SOLACE_TALENT,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-        cooldown: haste => 12 / (1 + haste),
+        cooldown: (haste:number) => 12 / (1 + haste),
```

Comment:
At the top of the file,
`spellbook(): SpellbookAbility[] {` 

Will both typecheck the abilities in the file and also automatically infer the `haste` type

Reply:
Fixed!